modules/atf_logger.lua

```diff
@@ -1,10 +1,37 @@
 local json = require("json")
 local Logger = {}
 Logger.mobile_log_format = "%s(%s) [version: %s, frameType: %s, encryption: %s, serviceType: %s, frameInfo: %s, messageId: %s] : %s \n"
-Logger.hmi_log_format = "%s(%s) : %s \n"
+Logger.hmi_log_format = "%s[%s] : %s \n"
+
+
+function capture_cmd_output(cmd, is_raw)
+  local f = assert(io.popen(cmd, 'r'))
```

Comment:
@LevchenkoS,
Maybe `os.date` could [handle](http://lua-users.org/wiki/OsLibraryTutorial) with that?